sway-lsp/tests/fixtures/tokens/abi/Forc.toml

```diff
@@ -3,6 +3,7 @@ authors = ["Fuel Labs <contact@fuel.sh>"]
 entry = "main.sw"
 license = "Apache-2.0"
 name = "abi"
+implicit-std = false
 
 [dependencies]
-std = { git = "https://github.com/FuelLabs/sway", tag = "v0.47.0" }
```

Comment:
Can we change these back to using the `tag = "v0.47.0"` for all of these?

Reply:
We can not.
The reason the CI wasn't working is because using that version of the std (which still contains the fixtures sources) confuses forc. The commit used is the first one where those are removed.